make_queue/models/models.py

```diff
@@ -115,6 +116,21 @@ class Status(models.TextChoices):
 
     STATUS_CHOICES_DICT = dict(Status.choices)
 
+    status = models.CharField(max_length=2, choices=Status.choices, verbose_name=_("Status"), default=Status.AVAILABLE)
+    stream_name = models.CharField(
+        max_length=30,
+        verbose_name=_("Stream Name"),
+        default=None, blank=True, null=True,
+        validators=[
+            RegexValidator(
+                regex="^[a-zA-Z0-9_-]+$",
+                message=_("Stream name can only consist of english letters, numbers, hyphens or underscores."),
+                code="NOT_URL_SAFE"
+            )
+        ]
+    )
```

Comment:
I don't remember precisely the convention but I think @ddabble introduced a code standard which distinguished between single and double quotation marks. Rougly, you use single in python files and double in the templates. @ddabble is it explicitly defined somewhere? Otherwise there has been som cleanup PRs that address this, like #350 I think.